src/test/java/org/aksw/iguana/commons/io/BigByteArrayInputStreamTest.java

```diff
@@ -0,0 +1,224 @@
+package org.aksw.iguana.commons.io;
+
+import com.google.common.primitives.Bytes;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Disabled("Big test")
```

Comment:
TODO: why is this one disabled?

Reply:
I'm not sure which of the BigByteArrayStreams it was, but one of the tests takes a long time (more than 2 minutes on my machine). I didn't want to run them each time I wanted to run all the tests, especially considering that I'm not touching these classes anymore. I will reable them soon, to test if they run on the GitHub machines.

Reply:
Let's keep this open then until that is clarified.

Reply:
doesn't run on GitHubActions